utils/stripe_utils.py

```diff
@@ -145,3 +218,18 @@ def make_payment(self, customer, amount, token):
             customer=customer
         )
         return charge
+
+    @staticmethod
+    def get_stripe_plan_id(cpu, ram, ssd, version, app):
```

Comment:
Please add "hdd" as a parameter as well already, even though it will be unused for the moment. I think right now it is easy to change and we might want to sell additional hdd space soon.